diff --git a/src/server/app/tabs/Media.py b/src/server/app/tabs/Media.py
index 5376781..a0f7e04 100644
--- a/src/server/app/tabs/Media.py
+++ b/src/server/app/tabs/Media.py
@@ -68,6 +68,6 @@ class Media(Tab):
                 engine.runAndWait()
                 subprocess.run([ "ffmpeg", "-i", nom_fichier, "-ar", "48000", "-ac",  "2", "-c:a", "pcm_s16le", f"{word}.wav" ])
                 os.remove(nom_fichier)
-            self.robot_connection.get_robot().play_audio(filename=f"{word}.wav").wait_for_completed()
+            self.ep_robot.play_audio(filename=f"{word}.wav").wait_for_completed()
         print("Lecture terminée.")
         return jsonify({"saying": data.get("say")})
diff --git a/src/server/app/tabs/RobomasterBasics.py b/src/server/app/tabs/RobomasterBasics.py
index 2a5fa9d..68d14f3 100644
--- a/src/server/app/tabs/RobomasterBasics.py
+++ b/src/server/app/tabs/RobomasterBasics.py
@@ -31,6 +31,7 @@ class RobomasterBasics(Tab):
         data = request.get_json()
         session[request.remote_addr] = SN[int(data.get("sn"))-1]
         response = self.robot_connection.connect(SN[int(data.get("sn"))-1])
+        super().ep_robot = self.robot_connection.get_robot(SN[int(data.get("sn"))-1])
         return jsonify({"start": response})
 
     def _stop(self):
diff --git a/src/server/lib/Tabs/Tab.py b/src/server/lib/Tabs/Tab.py
index 73d528e..8b832ae 100644
--- a/src/server/lib/Tabs/Tab.py
+++ b/src/server/lib/Tabs/Tab.py
@@ -10,7 +10,6 @@ class Tab:
 
     def __init__(self, robot_connection):
         self.robot_connection = robot_connection
-        self.ep_robot = self.robot_connection.get_robot()
         class_name = self.__class__.__name__.lower()
         for method_name in self.__class__.__dict__.keys():
 
